lib/semian/net_http.rb

```diff
@@ -44,7 +44,7 @@ def semian_identifier
     ].freeze # Net::HTTP can throw many different errors, this tries to capture most of them
 
     class << self
-      attr_accessor :exceptions
+      attr_accessor :exceptions, :disable_semian
```

Comment:
That's process global, so not thread safe. 

Would make more sense to have it on the instance or something like that.

Reply:
:nod: for sure